File: rust/src/models/billing_config.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, Default, PartialEq, Deserialize, Serialize)]
pub struct BillingConfig {
    #[serde(skip_serializing_if = "Option::is_none")]
    pub billing_cycles: Option<i32>,

    #[serde(skip_serializing_if = "Option::is_none")]
    pub net_terms: Option<i32>,

    #[serde(skip_serializing_if = "Option::is_none")]
    pub period_start_day: Option<i32>,
}

impl BillingConfig {
    pub fn new() -> Self {
        Self {
            billing_cycles: None,
            net_terms: None,
            period_start_day: None,
        }
    }
}

File: typescript/src/models/billingConfig.ts
// this file is @generated

export interface BillingConfig {
  billingCycles?: number | null;

  netTerms?: number;

  periodStartDay?: number | null;
}

export const BillingConfigSerializer = {
  _fromJsonObject(object: any): BillingConfig {
    return {
      billingCycles: object["billing_cycles"],
      netTerms: object["net_terms"],
      periodStartDay: object["period_start_day"],
    };
  },

  _toJsonObject(self: BillingConfig): any {
    return {
      billing_cycles: self.billingCycles,
      net_terms: self.netTerms,
      period_start_day: self.periodStartDay,
    };
  },
};
